src/ui/ui_render/views/chrome.jade

```diff
@@ -120,6 +117,6 @@ html(lang='en')
     //- good because we may use them to override EUI styles.
     style#themeCss
 
-  body(kbn-chrome, id='#{appName}-body')
-    kbn-initial-state(data=JSON.stringify(kibanaPayload))
+  body
+    kbn-injected-metadata(data=JSON.stringify(injectedMetadata))
```

Comment:
__nit:__ can you please also remove the last `kbn-initial-state` occurrence from https://github.com/elastic/kibana/blob/831b9f83fdfcf04a34a9896f899672c6c419fea8/src/core_plugins/console/public/tests/src/integration.test.js#L36 and update comment here https://github.com/elastic/kibana/blob/6d81bafe3d7179a4033496078914aaf962dacde0/src/core_plugins/state_session_storage_redirect/package.json#L4?